package.json

```diff
@@ -7,5 +7,8 @@
     "dependencies": {
         "express": "^4.17.1",
         "imba": "^2.0.0-alpha.146"
```

Comment:
The change for the RC version in `package-lock.json` is not reflected here - if we do want to update the RC version, I can bump it here too.